content/pages/photo-policy.md

```diff
@@ -0,0 +1,6 @@
+We take pictures at events, but won't take your picture if you're
+wearing a lanyard (We keep some lanyards available at all our events).
+
+We'll always tell everyone at the beginning of the event that photos
+taken will end up on social media unless you specifically ask otherwise or
+are wearing a lanyard.
```

Comment:
"If you have any questions about photography, email us at (email), or talk to one of the committee members at the event."